### DISCUSSION TITLE
Search in Tailwind V1 Documentation Broken

### DISCUSSION THREAD

**Discussion by kubik101** [tailwindlabs/tailwindcss#3571] [Help]
Just recently the search will not bring up any results to do with class types etc.
https://v1.tailwindcss.com/

**Comment by simonswiss** [comment]
Hey!

Just tried it, and seems to work fine for me. If that's still an issue, do you have more details (browser, OS, ...)?

**Reply by kubik101** [reply]
So make sure you are on V1 docs.
Search for some of the core things like:
- width
- height
- margin
- padding
- zindex

Exists on all browsers.
All on MacOSX Big Sur
Chrome: Version 88.0.4324.150 (Official Build) (x86_64)
Safari: Version 14.0.3 (16610.4.3.1.4)
Firefox: 85.0 (64-bit)
